build_manylinux_wheels.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Builds manylinux cyipopt wheels with Ipopt 3.14.11 based on MUMPS 5.5.1, and OpenBLAS 0.3.15
+set -eu  # Stop script if a line fails
+TAG=${1}
+echo "Building cyipopt with tag $TAG"
+
+# Install efficient BLAS & LAPACK library
+yum install -y openblas-devel-0.3.15-4.el8
+
+pushd /tmp
+
+# MUMPS (Linear solver used by Ipopt)
+git clone https://github.com/coin-or-tools/ThirdParty-Mumps --depth=1 --branch releases/3.0.4
+pushd ThirdParty-Mumps
+sh get.Mumps
+./configure --with-lapack="-L/usr/include/openblas -lopenblas"
+make
+make install
+popd
+
+
+# Ipopt (The solver itself)
+git clone https://github.com/coin-or/Ipopt --depth=1 --branch releases/3.14.11
```

Comment:
Why don't you just yum install ipopt? Seems like either 3.13 or 3.14 are available: https://packages.fedoraproject.org/pkgs/coin-or-Ipopt/coin-or-Ipopt/

Reply:
I see you say "but I would rather rely on building ipopt and mumps from source, so that we have recent versions".